apps/src/templates/InlineDropdownMenu.jsx

```diff
@@ -0,0 +1,143 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+import onClickOutside from 'react-onclickoutside';
+
+// HTML Adapted from:
+// https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-actions.html
+
+// Displays an accessible dropdown menu with actions or links
+// TODO: Convert to a functional component once Radium is upgraded past 0.26.0
+// Our version of Radium doesn't support useState, but it has been patched in
+// later versions
+export const InlineDropdownMenu = class InlineDropdownMenuComponent extends Component {
```

Comment:
two questions here:
1. why not name this component InlineDropdownMenu? InlineDropdownMenuComponent is never used so i think it's unnecessary unless i'm missing something
2. can we make this a functional component that uses hooks? does radium somehow prevent this?

Reply:
Hmm not sure if I remember why I did #1. I'll get back to you on that one.
#2: There's a pretty nasty bug in 0.25, which we're on, which causes radium to not work with hooks. It's been fixed in 0.26.0. Details: https://github.com/FormidableLabs/radium/issues/1016

Reply:
ahhh, i thought i saw a recent comment about radium+hooks -- thank you!